app/models/policy.rb

```diff
@@ -678,7 +678,7 @@ def terminate_as_of(term_date)
     self.aasm_state = "hbx_terminated"
     self.enrollees.each do |en|
       if en.coverage_end.blank? || (en.coverage_end.present? && (en.coverage_end > term_date))
-        en.coverage_end = term_date
+        en.coverage_end = (term_date > en.coverage_start ? en.coverage_start : term_date)
```

Comment:
It seems like the condition in the ternary operator might be reversed. If I understand correctly, we want to set en.coverage_end to term_date if term_date is greater than en.coverage_start. Otherwise, we want to set it to en.coverage_start. So, the correct code should be:
```
en.coverage_end = (term_date > en.coverage_start ? term_date : en.coverage_start)
```
Could you please confirm if my understanding is correct? Thanks!

Reply:
good catch